securesignals-ima-devapp/build.gradle

```diff
@@ -0,0 +1,52 @@
+plugins {
+    id 'com.android.application'
+    id 'org.jetbrains.kotlin.android'
+}
+
+apply from:  rootProject.file("$rootDir/common.gradle")
+
+android {
+    namespace 'com.uid2.dev'
+    compileSdk 33
+
+    defaultConfig {
+        applicationId "com.uid2.securesignals.ima.devapp"
+        minSdk 21
+        targetSdk 33
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            shrinkResources true
+            minifyEnabled true
+        }
+    }
+
+    lint {
+        disable 'GradleDependency', 'IconDipSize', 'IconDensities', 'RtlEnabled'
```

Comment:
Curious why the `IconX` disables are required?

Reply:
The [images that the Google provides](https://github.com/googleads/googleads-ima-android/tree/main/basicexample/app/src/main/res) cause the Linter to get hung up because of their sizing.  Since this is just a DevApp it doesn't hurt anything to ignore them at this point in time.

<img width="1419" alt="Screenshot 2023-04-19 at 10 26 51 AM" src="https://user-images.githubusercontent.com/989928/233124644-b46bfbc8-962d-4f68-a82e-16a0d15c41dc.png">